wordpress-mobile/WordPress-iOS
# Feature/7675 google button touch states

Refs #7675
Tweaks the google button so it shows a tapped state.  I opted to use light blue for the tapped state for consistency with hovers in Calypso and the behavior of link-style buttons in the iOS reader.

Normal:
<img width="256" alt="screen shot 2017-10-19 at 3 36 12 pm" src="https://user-images.githubusercontent.com/1435271/31796080-ce258460-b4ed-11e7-99a9-8abf6a15b0c3.png">

Pressed:
![simulator screen shot - iphone 6s - 2017-10-19 at 16 05 45](https://user-images.githubusercontent.com/1435271/31796088-d1ce35e4-b4ed-11e7-96ce-461bfc700ba0.png)

To test:
View the LoginEmailViewController and tap the button. Confirm that it changes state.

Needs review: @nheagy for code & @folletto for style
